src/org/aavso/tools/vstar/ui/dialog/plugin/manager/PluginManager.java

```diff
@@ -754,7 +759,12 @@ public void deletePlugin(String description) {
 					// vagaries of file systems or the possibility of
 					// concurrent deletion.
 					if (libJarPath.exists()) {
-						// if (libJarPath.delete()) {
+						// Max: This fragment was commented out by David(?)
```

Comment:
I'm wondering whether I commented it out because of a problem with deletion of files from WebStart launched VStar. Would need to search history. So long ago it may have been svn not git.